atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraExpiringKeyValueService.java

```diff
@@ -73,20 +73,14 @@ private CassandraExpiringKeyValueService(
 
     @Override
     public void put(TableReference tableRef, Map<Cell, byte[]> values, long timestamp, long time, TimeUnit unit) {
-        try {
-            putInternal(
-                    tableRef,
-                    KeyValueServices.toConstantTimestampValues(values.entrySet(), timestamp),
-                    CassandraKeyValueServices.convertTtl(time, unit));
-        } catch (Exception e) {
-            throw Throwables.throwUncheckedException(e);
-        }
+        expiringPut("put", tableRef, values, timestamp, time, unit);
     }
 
     @Override
     public void putWithTimestamps(TableReference tableRef, Multimap<Cell, Value> values, long time, TimeUnit unit) {
         try {
-            putInternal(tableRef, values.entries(), CassandraKeyValueServices.convertTtl(time, unit));
+            putInternal("putWithTimestamps", tableRef, values.entries(),
```

Comment:
should this also call `expiringPut`?

Reply:
I tried, but it actually have signatures (note that `expiringPut` adds the timestamp to the cells, where here we already have it).